appveyor.yml

```diff
@@ -0,0 +1,20 @@
+install:
+  - cmd: choco install wget
+  - cmd: wget https://github.com/libexpat/libexpat/releases/download/R_2_2_5/expat-win32bin-2.2.5.exe --output-document=C:\expat.exe
+  - cmd: C:\expat.exe /silent
+  - cmd: msbuild "Src\Win32\Dasher.sln"
+  - cmd: if exist Src\Win32\Target\%DASHERCONFIG%\Dasher.exe appveyor exit
+  - cmd: if not exist Src\Win32\Target\$%DASHERCONFIG%\Dasher.exe exit 1 
+
+environment:
+  expat: 'C:\Program Files (x86)\Expat 2.2.5'
+  GENERATOR: Visual Studio 15 2017
+  PLATFORM: Win32
+  APPVEYOR_BUILD_WORKER_IMAGE: Visual Studio 2017
+  matrix:
+    - configuration: Release
+      DASHERCONFIG: Release
```

Comment:
Is it possible to do only Release and leave the Dasher.wxs alone (except the commented out sections)?

When we only use Release here, and only check Target\Release\Dasher.exe, does Appveyor successfuly build? If that is the case, then let's do that leave the references to Release unchanged in Dasher.wxs.